src/main/scala/dpla/ingestion3/entries/ingest/JsonlEntry.scala

```diff
@@ -52,10 +75,9 @@ object JsonlEntry {
       }
     )
 
-    indexRecords.coalesce(1).write.text(outputName)
-
+    indexRecords.coalesce(1).write.json(dataOut)
```

Comment:
Unrelated question: Do we need this `coalesce` here? It probably slows things down quite a bit.

Reply:
Not sure but I thought we needed it to get all of the output into a single file?

Reply:
I'm assuming in asking this that we don't need a single file. Do we?

Reply:
I suppose if having multiple files isn't a negative then there is no reason to coalesce. But if we are indexing files by hand having to run elasticdump multiple times could be a real pain.

Reply:
Lookie here! https://www.npmjs.com/package/elasticdump#multielasticdump